package.json

```diff
@@ -116,6 +116,7 @@
     "jest-cli": "^24.9.0",
     "jest-environment-jsdom": "^24.9.0",
     "jest-environment-jsdom-global": "^1.2.0",
+    "jest-expo": "^37.0.0",
```

Comment:
b706f4d47 jest: Add `jest-expo` preset, to be used in the next commit.

This commit is in this PR (instead of a PR that doesn't have RN v0.61) because I didn't want to bother figuring out all the appropriate versions based on RN v0.60, which we'll move away from soon.

Reply:
One thing I haven't looked into further is that this particular version of jest-expo isn't compatible with Jest 26.1.0, which we're thinking of switching to soon (see [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/Lolex.20timers/near/927725)).